### PR TITLE
[General] show stage channels in `[p]serverinfo 1`

### PR DESCRIPTION
**Pull request by japandotorg**
### Description of the changes
Show the number of stage channels inside the Channel field in `[p]serverinfo 1`

### Have the changes in this PR been tested?
No

### CODE REVIEW

**Comment by Flame442** [comment]
This PR is currently failing the style check. This repo uses the Black autoformatter, and your changes need to be modified to comply with its specifications. Instructions on how to use it can be found in our [contributing.md file](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/CONTRIBUTING.md#43-style).

**Comment by japandotorg** [comment]
Oof looks like the style check failed again, i'll properly format it in 2-3 days, my exams are going on atm bc of which i'm kinda busy rn, i'll get back to this pr in a few days.

**Comment by japandotorg** [comment]
> This PR is currently failing the style check. This repo uses the Black autoformatter, and your changes need to be modified to comply with its specifications. Instructions on how to use it can be found in our [contributing.md file](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/CONTRIBUTING.md#43-style).

All done now, passed all the checks as well, ty ty

File: redbot/cogs/general/general.py [hunk 1]
```diff
@@ -410,7 +411,12 @@ def _bitsize(num: int):
                 value=_(
                     "\N{SPEECH BALLOON} Text: {text}\n"
                     "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}"
```
Comment (Flame442) [inline redbot/cogs/general/general.py]:
```suggestion
                    "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}\n"
```
Missing a newline here, to match the current formatting

**Review by Flame442 — CHANGES_REQUESTED** [review]
One little formatting issue, otherwise good to go! Thanks for contributing.

**Comment by japandotorg** [comment]
Whoops, guess I'm done now. Ty ty!

**Review by Flame442 — APPROVED** [review]

**Pull request merged**